src/App.tsx

```diff
@@ -17,15 +30,17 @@ export const App: React.FC = () => {
             <div className="control">
               <input
                 type="text"
+                value={query}
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => setQuery(event.target.value)}
```

Comment:
for my opinion better to create handler, and remove all logic from jsx

Reply:
I'm not sure, but maybe it's a good point when we have two and more strings. In this situation we can return value(update state) just in one string, and it's not necessarily to create a separate handler.

Reply:
You can leave it, but I think it will be much better if there is a minimum of logic in the jsx.

Reply:
@Bogdan-Mykhailov Good point